html_views/package.json

```diff
@@ -12,18 +12,13 @@
     "Programming Languages"
   ],
   "scripts": {
-    "compile": "gulp build",
+    "compile": "tsc -p src/goals && tsc -p src/ltacprof && shx mkdir -p ../client/html_views/{goals,ltacprof,jquery} && shx cp src/goals/*.{css,html} ../client/html_views/goals && shx cp src/ltacprof/*.html ../client/html_views/ltacprof  && shx cp jquery/**/*.js ../client/html_views/jquery",
     "postinstall": "typings install"
```

Comment:
CI [complains](//travis-ci.com/coq-community/vscoq/builds/119640252#L474) that:
> sh: 1: typings: not found

Since typings is deprecated in favor of NPM @types, we might need to change the postinstall script correspondingly.